src/cow-react/modules/limitOrders/utils/getDecimals.ts

```diff
@@ -0,0 +1,13 @@
+import { DEFAULT_DECIMALS } from '@src/custom/constants'
+import { Currency } from '@uniswap/sdk-core'
+
+export function getDecimals(currency: Currency): number {
+  const customMap: { [key: string]: number } = {
+    ALI: 18,
```

Comment:
I think it's a problem in the tokens list and we should fix it there

Reply:
It seems that way, https://www.gemini.com/uniswap/manifest.json on line `639` it has value of 8 decimals but I'm not sure who has control over this list, maybe Uniswap. So I would keep this as a temporary solution for now.